comparative-population-studies.csl

```diff
@@ -137,20 +137,9 @@
       <if variable="DOI">
         <text variable="DOI" prefix="[https://doi.org/" suffix="]"/>
       </if>
-      <else-if type="article-newspaper">
-        <group delimiter=", ">
-          <text variable="URL" prefix="[" suffix="]"/>
-          <date variable="accessed" delimiter=".">
-            <date-part name="day" form="numeric"/>
-            <date-part name="month" form="numeric"/>
-            <date-part name="year"/>
-          </date>
-        </group>
-      </else-if>
```

Comment:
Thanks for catching this. I got some conflicting ideas when discussing with the editor. I clarified and made a new revision.